app/js/views/services.jsx

```diff
@@ -69,7 +69,3 @@ export default class Services extends BaseView {
     );
   }
 }
-
-Services.propTypes = {
```

Comment:
Good catch. File an issue in their project and see what happens. If I understand correctly, it doesn't impact us right now.